### ISSUE TITLE
write different log levels to logfile and repl

### ISSUE THREAD

**Issue by mackenzieclark** [Delgan/loguru#640]
So, I love using loguru because it's simple to use and I don't really have my head wrapped around managing the default python logging. I don't write complex stuff, just scripts really.

Currently, I have a line like this at the start of most of my scripts:

`logger.add("outputs/logs/logfile_{time}.log")`

Then, I use different log levels for stuff, no configuring loguru, just using it out-of-the-box: `logger.info("cool stuff")` and `logger.debug("probably not very cool")` and `logger.error(e)` and so on.

Is there an easy, straightforward way do set things up so that, for example, log level `info` and `error` are written to the VScode interactive window, and then log level `debug`, `info`, and `error` are all written to the logfile?

Basically, I would like to log additional stuff for debugging, but I don't want it appearing in my REPL.

**Comment by Delgan** [comment]
You can configure the minimum threshold for messages to be logged using the `level` argument, or you can use a custom `filter` function.

```python
logger.add("outputs/logs/logfile_{time}.log")
logger.add(sys.stderr, level="INFO")  # Only "INFO" en above messages.
```

**Comment by mackenzieclark** [comment]
Thank you! This worked once I added `logger.remove()` one line before those two. This sort of quick, one-line change is exactly what I needed, I didn't want to fiddle with it much.

**Comment by Delgan** [comment]
Great! Is this issue solved and can I close it then?

**Comment by mackenzieclark** [comment]
Thank you, yes!
